Adds comment about data sampling time + general enhancements (#2639)
elastic/docs-content: deploy-manage/monitor/autoops/ec-autoops-events.md

    ess: all
products:
  - id: cloud-hosted
navigation_title: Events
---

# AutoOps events [ec-autoops-events]

> AutoOps continuously monitors your {{es}} deployments by sampling performance and health metrics at 10-second intervals. This high-frequency data collection allows for rapid detection and diagnosis of potential issues so you can get timely notifications and resolve issues faster. 

Reviewer: ```suggestion
AutoOps continuously monitors your {{es}} deployments by sampling performance and health metrics at 10-second intervals. This high-frequency data collection allows for timely notifications so you can rapidly detect, diagnose, and resolve potential issues. 
```
If this is accurate, cuts out some words

Author: Thank you! Actually, AutoOps detects and diagnoses issues and then notifies users so that they can resolve the issues, not the other way round. I think I'll stick with the original order but I'll shorten the sentence

When AutoOps detects an issue, it creates an event. Events provide detailed analyses of detected issues, including why they were triggered and the steps needed to resolve them. 

## Event insights

You can view events on the **Deployment** page in the **Open Events** and **Events History** sections.

> When you select an event, a flyout appears with insights and context around the detected issue.

Reviewer: ```suggestion
When you select an event, a flyout appears with insights and context for the detected issue.
```

:::{image} /deploy-manage/images/cloud-autoops-events.png
:alt: AutoOps events
:::

> The following table describes the different sections in this flyout:

Reviewer: ```suggestion
The following table describes the flyout's sections:
```

| Section | Description |
| --- | --- |
| What was detected | Describes why the event was created and provides links to drill down into the detected issue. |
| Recommendations | Lists recommendations to address the issue and improve your cluster's overall performance. The recommendations are organized according to the suggested order of execution. |

> | Event duration | Shows the time the event was created when AutoOps detected the issue, and if applicable, the time the event was closed when AutoOps identified that the issue no longer exists. The closing of an event doesn't necessarily mean that the issue is resolved, just that AutoOps no longer detects it. |

Reviewer: ```suggestion
| Event duration | Shows when the event was created (when AutoOps detected the issue), and if applicable, when the event was closed (when AutoOps identified that the issue no longer exists). A closed event doesn't necessarily mean the issue is resolved, just that AutoOps no longer detects it. |
```
I was a bit confused on first read, hopefully the parenthesis and using "when" instead of "the time" makes it a bit more clear for noobs such as myself.

Author: Thanks! This looks much better
